app.config.ts

```diff
@@ -45,9 +41,9 @@ const createConfig = (): Omit<ExpoConfig, 'extra'> & { extra: { eas: EASConfig }
     },
     orientation: 'portrait',
     backgroundColor: '#000000',
-    icon: './src/assets/images/icon.png',
+    icon: './assets/images/icon.png',
```

Comment:
Shouldn't the icon and other assets in `/assets` be moved to `/libs/shared/ui/assets` as described in the latest changes of our documentation